config/locales/en.yml

```diff
@@ -0,0 +1,15 @@
+---
+en:
+  decidim:
+    authorization_handlers:
+      members_picker_authorization_handler:
+        explanation: Pick which emails are allowed to participate from the component
+          permissions screen
+        fields:
+          members_picker: Allowed members' email
+          members_picker_help: "(separated by commas)"
+        name: Pick which members' emails can participate by component
```

Comment:
Maybe it is better "Pick the emails of the  users that will be able to participate in the current component"?

Reply:
ok